lib/rex/proto/kerberos/model/enc_krb_cred_part.rb

```diff
@@ -46,6 +60,50 @@ def encode_ticket_info
             encoded = ticket_info.map {|t| t.encode }
             seq = OpenSSL::ASN1::Sequence.new(encoded)
           end
+
+          # Decodes a Rex::Proto::Kerberos::Model::EncKrbCredPart from an String
+          #
+          # @param input [String] the input to decode from
+          def decode_string(input)
+            asn1 = OpenSSL::ASN1.decode(input)
+
+            decode_asn1(asn1)
+          end
+
+          # Decodes a Rex::Proto::Kerberos::Model::EncKrbCredPart
+          #
+          # @param input [OpenSSL::ASN1::ASN1Data] the input to decode from
+          # @raise [Rex::Proto::Kerberos::Model::Error::KerberosDecodingError] if decoding doesn't succeed
+          # ticket-info     [0] SEQUENCE OF KrbCredInfo,
+          # nonce           [1] UInt32 OPTIONAL,
+          # timestamp       [2] KerberosTime OPTIONAL,
+          # usec            [3] Microseconds OPTIONAL,
+          # s-address       [4] HostAddress OPTIONAL,
+          # r-address       [5] HostAddress OPTIONAL
```

Comment:
I think this will get grouped by yard doc into the `@raise` information, I don't believe that's intentional

Reply:
ah yea it is not